code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -18,11 +18,14 @@
 //Hunter verbs
 
 /mob/living/carbon/alien/humanoid/hunter/proc/toggle_leap(message = 1)
+	if(pounce_cooldown)

```

Comment:
I did that intentionally. When I was playing it was confusing being able to turn on the toggle but not use it. I want it to work like spells where you have immediate feedback that its not ready rather than entering the stance/lighting up the toggle and having it not work anyway.